code/modules/paperwork/records/recordsfiling.dm

```diff
@@ -111,3 +111,30 @@
 		virgin = FALSE
 	return ..()
 
+
+/*
+ * Lore Cabinet
+ */
+
+//Zayin
+/obj/structure/filingcabinet/zayinlore
+	name = "zayin supplimentary information cabinet"
+	icon_state = "employmentcabinet"
+	var/virgin = TRUE
+
+/obj/structure/filingcabinet/zayinlore/proc/fillCurrent()
```

Comment:
```suggestion
/obj/structure/filingcabinet/zayinlore/Initialize()
```
Either that, or actually add a proc call for the FillCurrent().

Reply:
It does call fill current, first time you attack hand, and it's why the virgin var is needed, also removes a lot of "Spawn 200 papers inside cabinets" on startup
Using initialize causes integration errors without giving me logs.